GH-132445: Allowing to reset parameters of Wave_write (#132448)
python/cpython: Doc/library/wave.rst

      Set the number of channels.

>       Raises an :exc:`wave.Error`, if the value is set after data was written.

Reviewer: ```suggestion
      Raise :exc:`wave.Error` if a different value is set after data was written.
```

>       .. versionchanged:: 3.14
>          :exc:`wave.Error` is not raised, if the value is the same.

Reviewer: ```suggestion
      .. versionchanged:: next
         No :exc:`wave.Error` is raised if the value does not change.
```

Same for the rest of the docs.
